DcCore/DcCore/Views/InitialsBadge.swift

```diff
@@ -112,7 +112,7 @@ public class InitialsBadge: UIView {
         return !label.isHidden
     }
 
-    public func setColor(_ color: UIColor) {
+    public func setColor(_ color: UIColor?) {
```

Comment:
color does not need to to optional, i have already changed this, no further action needed :)